src/dashboards/Compact/ScooterTile/index.tsx

```diff
@@ -41,15 +32,15 @@ function ScooterTile({ scooters }: Props): JSX.Element {
                     latitude={viewport.latitude || 0}
                     longitude={viewport.longitude || 0}
                 >
-                    <PositionPin size="24px" />
+                    <PositionPin size="24" />
```

Comment:
Meiner du `size={24}` her, for å få det som `number`?

Reply:
Jepp 🙈